Voyage/Voyage.Common/Enums/Color.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Voyage.Common.Enums
+{
+    public enum Color
+    {
+        White = 0,
+        Red = 1,
+        Green = 2,
+        Blue = 3,
+        Black = 4,
+        Grey = 5,
+        Yellow = 6,
+        Purple = 7,
```

Comment:
There can be problem. What if driver has whitesmoke color of tansport? Or another unusual variant?